docs/src/main/sphinx/connector/hive.rst

```diff
@@ -105,6 +105,11 @@ modes.
 
 You can configure the behavior in your catalog properties file.
 
+By default, Hive views are executed with the ``RUN AS DEFINER`` security
+configuration set with the ``hive.hive-views.run-as-invoker`` catalog
+configuration property. You can change the value to ``RUN AS INVOKER`` for the
```

Comment:
The property accepts true / false. One suggestion would be "You can change the value to ``true`` to have Hive views executed with the ``RUN AS INVOKER`` security configuration instead.